packages/components/pie-button/tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Would be good to pair on this just for an overview – I have no idea what any of this means 🤣

Reply:
Ill be interested too.

Reply:
I've created a ticket for extracting the TypeScript config, so I think part of the process will be working out what everything means and what is required 😄